.ci/jobs.t/elastic+helm-charts+{branch}+staging+cluster-cleanup.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - job:
-    name: elastic+helm-charts+staging+cluster-cleanup
+    name: elastic+helm-charts+%BRANCH%+staging+cluster-cleanup
```

Comment:
Staging jobs where created by #532 to tests staging Docker image. IMHO these jobs should stay only on master branch.

Reply:
With this config we should automatically have staging jobs generated per git branch. Those "template" files will be present on all git branches (without needed to update them)